cmake/Bond.cmake

```diff
@@ -129,6 +130,18 @@ function (add_bond_test test)
             WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
             COMMAND ${test})
     endif()
+
+    get_property (lastTestPort GLOBAL PROPERTY LAST_TEST_PORT)
+    if (NOT DEFINED lastTestPort)
+        set (lastTestPort 25189)
+    endif()
+    math(EXPR testPort1 "${lastTestPort} + 1")
+    math(EXPR testPort2 "${lastTestPort} + 2")
+    target_compile_definitions (${test} PRIVATE -DTEST_PORT_1=${testPort1})
+    target_compile_definitions (${test} PRIVATE -DTEST_PORT_2=${testPort2})
+    target_compile_definitions (${test} PRIVATE -DEXAMPLE_PORT_1=${testPort1})
```

Comment:
`TEST_PORT1` seems an OK name, even for examples. That way we only have two of these things, not four.